package.json

```diff
@@ -48,6 +48,7 @@
     "react-dom": "^16.11.0",
     "react-helmet": "^5.2.0",
     "react-hook-inview": "^4.1.2",
+    "react-icons": "^3.8.0",
```

Comment:
I added react icons because font awesome does not provide a thin version for the downward facing arrow in the free version.